content/docs/csidriver/installation/helm/unity.md

```diff
@@ -107,13 +107,17 @@ Procedure
       * ArrayId corresponds to the serial number of Unity XT array.
       * Unity XT Array username must have role as Storage Administrator to be able to perform CRUD operations.
       * If the user is using a complex K8s version like "v1.24.6-mirantis-1", use this kubeVersion check in helm/csi-unity/Chart.yaml file.
-            kubeVersion: ">= 1.24.0-0 < 1.28.0-0"
+            kubeVersion: ">= 1.24.0-0 < 1.29.0-0"
 
-2. Copy the `helm/csi-unity/values.yaml` into a file named `myvalues.yaml` in the same directory of `csi-install.sh`, to customize settings for installation.
+2. Get the required values.yaml using below command
```

Comment:
using the command below
or 
using the following command

Reply:
Taken care

Reply:
@sharont58  Could you please review and approve this PR ?